unit/widgets/button_spec.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+require 'rspec'
+#require 'byebug'
+#require 'ruby-debug'
+require_relative '../../../lib/libyui_client/http/response'
+
+
+module LibyuiClient
+  module Widgets
+    RSpec.describe Button do
+      describe '.Button' do
+        
+        context 'when be initialized' do
+          it 'should be a Button instance with default timeout and interval' do
+            widget_controller = double('widget_controller')
+            widget_filter = double()
+            @button = Button.new(widget_controller, widget_filter)
```

Comment:
This is repeated in the other examples. Ideally it will move to the before hook at the end

Reply:
because those are "cosmetic in some way i am leaving that for the end. i commented to avoid get comments for such stuff from this early state